app/controllers/users_controller.rb

```diff
@@ -0,0 +1,44 @@
+class UsersController < ApplicationController
+  def show
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to user_path(@user)
+    else
+      respond_with(@user)
+    end
+  end
+
+  def edit
+  end
+
+  def update
+    if user.update(user_params)
+      redirect_to user_path(user)
+    else
+      respond_with(user)
+    end
+  end
+
+  def destroy
+    user.delete
+    redirect_to root_path
```

Comment:
Newline break?

Reply:
line break in, plus a refactor, thanks Elle